tools/harness-simulation/README.md

```diff
@@ -10,15 +10,33 @@ Platform developers should modify the THCI implementation and/or the SI implemen
 
```

Comment:
Can you change line 7 from "So both two systems" to "So two systems" please?  I can't comment on lines outside of the changed portion of the file.